src/Illuminate/Cache/Lock.php

```diff
@@ -58,6 +58,14 @@ public function __construct($name, $seconds, $owner = null)
      */
     abstract public function acquire();
 
+    /**
+     * Extend the lock.
+     *
+     * @param int $seconds
+     * @return bool
+     */
+    abstract public function extend($seconds);
```

Comment:
This is a major breaking change, so this PR will have to target the master branch.

Reply:
Sorry, missed that. Updated to master.